src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -536,6 +536,48 @@ static inline void dma_set_irq1_channel_mask_enabled(uint32_t channel_mask, bool
     }
 }
 
+/*! \brief  Determine if a particular channel is a cause of DMA_IRQ_0
+ *  \ingroup hardware_dma
+ *
+ * \param channel DMA channel
+ * \return true if the channel is a cause of DMA_IRQ_0, false otherwise
+ */
+static inline bool dma_channel_get_irq0_status(uint channel) {
+    check_dma_channel_param(channel);
+    return dma_hw->ints0 & (1u << channel);
+}
+
+/*! \brief  Determine if a particular channel is a cause of DMA_IRQ_1
+ *  \ingroup hardware_dma
+ *
+ * \param channel DMA channel
+ * \return true if the channel is a cause of DMA_IRQ_1, false otherwise
+ */
+static inline bool dma_channel_get_irq1_status(uint channel) {
+    check_dma_channel_param(channel);
+    return dma_hw->ints0 & (1u << channel);
+}
+
+/*! \brief  Acknowledge a channel IRQ, resetting it as the cause of DMA_IRQ_0
+ *  \ingroup hardware_dma
+ *
+ * \param channel DMA channel
+ */
+static inline void dma_channel_acknowledge_irq0(uint channel) {
+    check_dma_channel_param(channel);
+    hw_set_bits(&dma_hw->ints0, (1u << channel));
+}
+
+/*! \brief  Acknowledge a channel IRQ, resetting it as the cause of DMA_IRQ_0
```

Comment:
DMA_IRQ_0 -> DMA_IRQ_1

Reply:
**bump**
This is the docstring for `dma_channel_acknowledge_irq1` so it should probably be talking about `DMA_IRQ_1` rather than `DMA_IRQ_0` ?

Reply:
This is marked as resolved, but doesn't seem to be fixed?

Reply:
causality mishaps - resolving as I go i guess i will leeave them open and you can resolve

Reply:
(i haven't committed yet)

Reply:
Ahhh, if you've already fixed these locally and just haven't committed yet then that's fine. Apologies if I was being over-zealous.